app/controllers/FileUploadController.scala

```diff
@@ -84,8 +85,13 @@ class FileUploadController @Inject()(val mcc: MessagesControllerComponents,
         } yield {
           file match {
             case Some(file: UploadedSuccessfully) =>
-              ersUtil.cache[String](ersUtil.FILE_NAME_CACHE, file.name, requestObject.getSchemeReference).map { _ =>
-                Redirect(routes.FileUploadController.validationResults())
+              if(file.name.contains(".csv")) {
+                Logger.warn("[FileUploadController][success] User uploaded a csv file instead of an ods file")
```

Comment:
I see the value in explicitly logging this case, but: won't this kind of logic mean that only a .csv is explicitly handled, and e.g. a .txt is allowed (goes into the 'else' case)? or is this handled higher up?

Reply:
Good point, think I was a little tunnel visioned here. I think because the other file types will just be caught by the file validation and subsequently be taken to 'file has errors'. 

I'll change it so it checks that it is explicitly an .ods file.